### ISSUE TITLE
Accordion: missing focus frame & some keyboard functionality

### ISSUE THREAD

**Issue by jkcox** [ASU-KE/UDS-WordPress-Theme#608]
https://wordpress.asu.edu/accessibility/uds-components/

The accordion component is missing a couple standard functionalities. Refer to ASU accordion component page for correct behavior: https://asu.github.io/asu-unity-stack/@asu/unity-bootstrap-theme/index.html?path=/story/atoms-accordions-templates--accordion 

Space bar doesn’t expand/collapse the accordions, but it should (currently, only the Enter/Return key expands/collapses it, which is good, but shouldn’t be the only method).

The focus frame is missing. Thus, when keyboard dependent user Tab navigates to the accordion, they’re unaware if the accordion is focused because the focus frame (black rectangle) is missing.
